spec/requests/api/v1/docs/display_ads_spec.rb

```diff
@@ -57,6 +57,14 @@
             organization_id: { type: :integer, description: "Identifies the organization to which the ad belongs" },
             display_to: { type: :string, enum: DisplayAd.display_tos.keys, default: "all",
                           description: "Potentially limits visitors to whom the ad is visible" },
+            type_of: { type: :string, enum: DisplayAd.type_ofs.keys, default: "in_house",
+                       description: <<~DESCRIBE
+                         Types of the billboads:
```

Comment:
There's a minor typo here:

```suggestion
                         Types of the billboards:
```